CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+
+- Governance actions currently staged for enactment
+- Governance actions under ratification, with the total and percentage of yes stake, no stake and abstain stake
+- The current constitutional committee, and constitution hash digest
+
+#### Bootstrapping Phase
+
+We will need to be careful how we bootstrap this fledgling government.  All the parties
+that are involved will need ample time to register themselves and to become familiar with the process.
+
+Special provisions will apply in the initial bootstrap phase.
+Firstly, during the bootstrap phase, a quorum vote from the constitutional committee
+is sufficient to change the protocol parameters.
+Secondly, during the bootstrap phase, a quorum vote from the constitutional committee,
+together with a sufficient SPO vote, is sufficient to initiate a hard fork.
+No other actions are possible during the bootstrap phase.
+
+The bootstrap phase ends when **either** of the following occurs:
+1. a sufficient amount of stake is delegated to DReps
+2. a given number of epochs has passed (likely to be a number of months after the hard fork)
+
+#### Final safety measure, post bootstrapping
+
+Many people have stated that they believe that the actual voting turnout will not be so large
+as to be a strain on the throughput of the system.
+We also believe that this is likely to be the case, but when the bootstrap phase ends we will
+put one final, temporary safety	measure in place (this will also allow us to justify a low DRep deposit amount).
+
+For values of $X$ and $Y$ that are still to be determined,
+as soon as the bootstrap phase has ended,
+when we calculate the DReps stake distribution for the next epoch boundary,
+we will consider _only_ those DReps that are _either_ in the top $X$-many DReps ranked by stake amount,
+or those DReps that have at least $Y$ Lovelace.
+Every epoch, the value of $X$ will _increase_ and the value of $Y$ will decrease,
+so that eventually $X$ will be effectively infinite and $Y$ will be zero.
+Note that this is only an incentive, and nothing actually stops any DRep from casting their
+vote (though it will not be counted if it does not meet the requirements).
+
+If the community decides at some point that there is indeed a problem with congestion,
+then a hard fork could be enacted that limits the number of DReps in a more restrictive way.
+
+Reasonable numbers for the initial value of $X$ are probably 5,000-10,000.
+Reasonable numbers for the initial value of $Y$ are probably the total number of Lovelace
+divided by the initial value of $X$.
+
+The mechanism should be set to relax at a rate where the restriction is completely eliminated after
+a period of six months to one year.
+
+#### Other Ideas / Open Questions
+
+##### Pledge-weighted SPO voting
+
+The SPO vote could additionally be weighted by each SPO's pledge.
```

Comment:
I believe this(Pledge-weighted SPO voting power) is a more candid approach for setting the voting power of an SPO, than using their consensus stake delegation. 

It properly separates them from a registered dRep.

The current approach seems like they're able to double-dip their voting power between stake pool weight and dRep weight, if SPO does become a dRep too.(correct me if I'm misinterpreting)

Besides, an SPO can still register to be a dRep and then properly actualise the remainder of the stake-weight of their voting delegators(if they choose to delegate their voting power to said SPO). 

So regular SPO: Voting Power from Pledged Stake(Consensus)

SPO who is also a registered dRep: Voting Power from Pledged Stake(Consensus) + Voting Power from Delegated Voting Power(Governance)

This will also encourage them to increase their pledge. 

I will also add that much like how the Pledge is meant to be met for the remaining time after a change i.e. two epochs I believe(Stake Pool experts please double check me on this), the same should apply for an SPO's voting capacity, lest their vote power be nullified. Perhaps the duration can be longer(anywhere from 3-6 epochs). This will also mitigate funny cases where pledge stake is only introduced for short-term votes and quickly dumped.

Reply:
I would imagine that only "active" pledge at the time of the voting period ending would count, so the "3 Epoch Rule" would apply to an SPO attempting to "game" the vote by briefly increasing their pledge before dropping it back down again. Any other period would seem arbitrary (i.e. one or two extra epochs)